src/components/fieldHelpers/label.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC } from "react"
+
+interface Props {
+  label?: string
+  required?: boolean
+  htmlFor?: string
+}
+
+const Required: FC = () => (
+  <span className="text-salmon leading-xxs relative top-requiredIndicator">
+    &nbsp;*
+  </span>
+)
+
+const Label: FC<Props> = ({ label, required = false, htmlFor }) => {
+  if (!label) return null
+  htmlFor = htmlFor || label
```

Comment:
What's the motivation behind this? Wouldn't this result in a translated string being set as `for` attribute?

Reply:
I don't think we need it actually. I used a label component from DH and did not clean enough !